#614 - Update WalletSelectScreen to include selfie image in the top left corner

brntsw opened the pull request:
Implemented the selfie on the top left corner of the Wallet selection screen. Also added the user's image on that selfie and also on the user selection screen.

A demo of how that's behaving:

https://user-images.githubusercontent.com/7141687/120409660-c5868380-c327-11eb-9bfe-8a36b458eeb2.mov

Elforama reviewed (changes requested):
This looks great! Just wondering if we still need the Box around the localimage. It looks like all those modifiers can be moved to the image instead.

brntsw commented:
I have removed the Box, it was really just a placeholder for the Image, I didn't realize that.
Also, I added the ContentScale parameter to LocalImage, as it was necessary to me to have that and I thought that would be unnecessary to create another Composable for that reason.

The behavior keeps the same:

https://user-images.githubusercontent.com/7141687/120472745-a6fea780-c37c-11eb-9adc-62d06a8742f1.mov

Elforama reviewed (approved):
Thanks for the quick update, great work!
